local/overlays/save_version.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
I am getting mismaches with the utilities files... When installing device with  for example: `python3 dcs_deploy.py flash orin_nx 512 2.0 "default" nvme full` 

I dont have any utilities installed in `/home/dcs_user/av_utilities` however `dcs_deploy_data.json` does list them as installed 
```
  "binaries": [
    "/home/dcs_user/av_utilities/fmu_cube_usb_switch.py",
    "/home/dcs_user/av_utilities/wifi_generator.py",
```

Reply:
This looks fine, i will re-test this

Reply:
Tested, works correctly.